week_5/assignment/lottery-api/src/app.service.ts

```diff
@@ -1,8 +1,40 @@
 import { Injectable } from '@nestjs/common';
+import { ethers } from 'ethers';
+import * as tokenJson from './assets/MyToken.json'; // TODO
+import { Lottery } from '../lottery-contract/contracts/Lottery.sol'; // Not sure why the error
```

Comment:
because we don't have Lottery.sol in this project. Instead of doing new Lottery.sol we will have to attach it with the contract on chain.